solvers/condat_linear.py

```diff
@@ -0,0 +1,39 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+from benchopt.stopping_criterion import SingleRunCriterion
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from benchmark_utils.tv_numba import linearized_taut_string
+
+
+class Solver(BaseSolver):
```

Comment:
I would not add such a solver but a PGD solver based on it would make a lot of sense. This is very similar to the current PGD solver we have that already uses the taut string algorithm for its prox, but there C implementation it relies on is not maintained anymore and does not work for py39+ IIRC

Reply:
May be a solution would be to put the prox implementation as a parameter of the PGD solver ?

Reply:
yes I think this would be nice indeed

Reply:
It's what we did here https://github.com/benchopt/benchmark_slope/blob/main/solvers/python_pgd.py#L15

Reply:
In this case, should we still have the 3 independent solvers? For me yes, as it shows their raw performance.

Reply:
I would say we can remove them as they don't tackle the same problem and TV denoising in 1D is not really of interest to many as there is a finite step algorithm.